content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -283,7 +298,7 @@ In practice, these are the steps necessary to introspect an existing DB and gene
 
 ```prisma
 datasource db {
-  provider = "postgresql"
+  provider = "postgres"
```

Comment:
Default is still `postgresql`: https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference/#datasource